pkg/vpcmodel/vpcConnectivity.go

```diff
@@ -23,20 +23,13 @@ type VPCConnectivity struct {
 	// This is auxiliary computation based on which AllowedConnsCombined is computed, however the "debug" format uses it
 	AllowedConns map[Node]*ConnectivityResult
 
-	// combined connectivity - considering both ingress and egress per connection
-	// The main outcome of the computation of which most of the outputs are based
-	// (outputs excluding json and debug)
-	// For each src node provides a map of dsts and the connection it has to these dsts, including stateful attributes
-	// a connection is considered stateful if all paths in it are stateful
-	// that stateful component is computed along with the following  AllowedConnsCombinedStateful
-	AllowedConnsCombined GeneralConnectivityMap
-
 	// allowed connectivity combined and stateful
 	// used by debug and json format only (at the moment)
 	// For src node provides a map of dsts and the stateful connection it has to these dsts
 	// note that subset of a non-stateful connection from AllowedConnsCombined can still be stateful
 	// and as such add to this map
-	AllowedConnsCombinedStateful GeneralConnectivityMap
+
+	AllowedConnsCombinedStateful GeneralStatefulConnectivityMap
```

Comment:
please open an issue to consider enhancing current connectivity reports with more accurate statefulness info, as available at this new type. (as opposed to only json that was adding this so far) .

Reply:
you mean two issues - one for connectivity and one for diffs. (We can prioritize each)

Reply:
or two tasks in the same issue

Reply:
#615 and #614 
I've put them in separate issue so that they could be assigned to e.g. different milestone